app/assets/stylesheets/pageflow/themes/legacy/overview.scss

```diff
@@ -0,0 +1,48 @@
+.overview {
+  @import "pageflow/themes/default/overview/chapter_border_images";
+
+  font-family: sourcesanspro;
+
+  &:after {
+    background-color: black;
+  }
+
+  &.active:after {
+    opacity: 0.4;
+  }
+
+  .content {
+    color: #fff;
+
+    h2 {
+      font-size: 3em;
+    }
+
+    .ov_chapter {
+      .heading-4 {
```

Comment:
Selector should have depth of applicability no greater than 3, but was 4<br>Selector `heading-4` should be written in lowercase with underscores